src/Http/WebUtilities/src/Microsoft.AspNetCore.WebUtilities.csproj

```diff
@@ -13,6 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="$(SharedSourceRoot)QueryStringEnumerable.cs" />
```

Comment:
The reason for shared-source is that, in a PR coming very soon to a repo near you, I'll be using `QueryStringEnumerable` as an internal API inside `Microsoft.AspNetCore.Components`. That package does not have a dependency on `WebUtilities`, and doesn't want to have one, since it's not web-specific and most of the baggage brought with `WebUtilities` would not be applicable.